private/bufpkg/bufcas/bufcasalpha/bufcasalpha.go

```diff
@@ -13,83 +13,113 @@
 // limitations under the License.
 
 // Package bufcasalpha temporarily converts v1alpha1 API types to new API types.
-//
-// Minimal validation is done as this is assumed to be done by the bufcas package,
-// and this allows us to have less error returning.
 package bufcasalpha
 
 import (
-	storagev1beta1 "buf.build/gen/go/bufbuild/registry/protocolbuffers/go/buf/registry/storage/v1beta1"
+	"bytes"
+
+	"github.com/bufbuild/buf/private/bufpkg/bufcas"
 	modulev1alpha1 "github.com/bufbuild/buf/private/gen/proto/go/buf/alpha/module/v1alpha1"
 )
 
 var (
-	digestTypeToAlpha = map[storagev1beta1.Digest_Type]modulev1alpha1.DigestType{
-		storagev1beta1.Digest_TYPE_SHAKE256: modulev1alpha1.DigestType_DIGEST_TYPE_SHAKE256,
+	digestTypeToAlpha = map[bufcas.DigestType]modulev1alpha1.DigestType{
```

Comment:
Updated this package to convert between the proto v1alpha1 types and the interface types in bufcas (instead of before where it was solely to map between v1alpha1 and v1beta1 types).

This package isn't needed once we land the bufmod branch but will be needed in the meantime.